.github/workflows/Manager-CI.yaml

```diff
@@ -150,6 +150,7 @@ jobs:
           SLACK_WEBHOOK_TOKEN: ${{ secrets.SLACK_WEBHOOK_TOKEN }}
           SLACK_CHANNEL: ${{ secrets.SLACK_CHANNEL }}
           WEBHOOK_SITE_UUID: ${{ secrets.WEBHOOK_SITE_UUID }}
+          WEBHOOK_SIDE_UUID_UPDATE_PROCESSOR_E2E: ${{ secrets.WEBHOOK_SIDE_UUID_UPDATE_PROCESSOR_E2E }}
```

Comment:
That webhook site could be used also in other tests, so I would name it rather to something like `WEBHOOK_SITE_UUID_SECOND` or in similar way.

Reply:
Okk, in that case I guess it could be updated.